### DISCUSSION TITLE
TS0201 humunity value change

### DISCUSSION THREAD

**Discussion by KlausV66** [Koenkk/zigbee2mqtt#20576] [Devices]
Hi, 
I bought the sensor [TS0201](https://www.zigbee2mqtt.io/devices/TS0201.html#tuya-ts0201) and I would like to calibrate the humunity value on the display. I changed the value by -10 and on HA it's correct, but the display value still wrong.
Is there a possibility to change?
Thanks.

**Comment by Looney1976** [comment]
Only if you're able to get into some sort of calibration status on the device itself are you able to change the readout value on the screen. Other than that, if it's a cheap sensor like the DHT sorts, they can be all over the place with regards to accuracy. If you want or need accurate humdity readings, I suggest you look into more expensive digital sensors.
